src/polymer/call-super-in-callbacks.ts

```diff
@@ -39,9 +39,10 @@ class CallSuperInCallbacks extends Rule {
   async check(document: Document) {
     const warnings: Warning[] = [];
 
-    const elementLikes =
-        Array.from(document.getFeatures({kind: 'element'}))
-            .concat(Array.from(document.getFeatures({kind: 'element-mixin'})));
+    const elementLikes = new Array<Element|ElementMixin>(
+        ...document.getFeatures({kind: 'element'}),
```

Comment:
consider changing this to just `'class'`, here and elsewhere

ok to punt this as well though.